### PR TITLE
Check to make sure selected WCs agree

### PR DESCRIPTION
**Pull request by kmohrman**
Add a check to the data card maker wrapper to raise and error if the list of selected WCs does not match the reference. Note this is not really the best way to handle this check (would be better to check against the one in the EFTFit repo, but that's harder) but at least should be better than nothing.

### CODE REVIEW

File: topcoffea/modules/utils.py [hunk 1]
```diff
@@ -162,3 +162,27 @@ def get_hist_from_pkl(path_to_pkl,allow_empty=True):
...
+# Check if the contents of two dictionaries of lists agree
+# Assumes structure d = {k1: [i1,i2,...], ...}
+def dict_comp(in_dict1,in_dict2,strict=False):
+
+    def all_d1_in_d2(d1,d2):
+        agree = True
+        for k1,v1 in d1.items():
+            if k1 not in d2:
+                agree = False
+                continue
```
Comment (bryates) [inline topcoffea/modules/utils.py:175]: Probably won't affect performance, but you could put `break` here, or even `return False`.

File: topcoffea/modules/utils.py [hunk 2]
```diff
@@ -162,3 +162,27 @@ def get_hist_from_pkl(path_to_pkl,allow_empty=True):
...
+            for i1 in v1:
+                if i1 not in d2[k1]:
+                    agree = False
```
Comment (bryates) [inline topcoffea/modules/utils.py:178]: Same here

**Review by bryates — APPROVED** [review]
Looks good. I had a small comment that doesn't need to be added.

File: topcoffea/modules/utils.py (hunk 1 above)
Comment (kmohrman) [reply topcoffea/modules/utils.py]: Thanks, added a `break`.

File: topcoffea/modules/utils.py (hunk 2 above)
Comment (kmohrman) [reply topcoffea/modules/utils.py:178]: Thanks, added a `break`.

**Pull request merged**